tests/expressions/tests.py

```diff
@@ -576,6 +576,14 @@ def test_nested_subquery_outer_ref_with_autofield(self):
         # This exercises the double OuterRef form with AutoField as pk.
         self.assertCountEqual(outer, [first, second])
 
+    def test_nested_subquery_outer_ref_with_operator(self):
+        first = Time.objects.create(time='09:00')
+        second = Time.objects.create(time='17:00')
+        inner = Time.objects.filter(time__lt=OuterRef(OuterRef('time')) * 2).values('time')
+        middle = SimulationRun.objects.annotate(other=Subquery(inner)).values('other')[:1]
+        outer = Time.objects.annotate(other=Subquery(middle, output_field=IntegerField()))
+        self.assertCountEqual(outer, [first, second])
```

Comment:
Can we also add a test for reported use case?

Reply:
I think this test can be entirely removed if `test_nested_subquery_outer_ref_with_function` covers the usage case appropriately.

Reply:
Agreed, removed.